sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/report/ResourceReport.java

```diff
@@ -40,13 +41,13 @@
 public final class ResourceReport {
   private final InputComponent component;
   private final IssueVariation total = new IssueVariation();
-  private final Map<ReportRuleKey, RuleReport> ruleReportByRuleKey = Maps.newHashMap();
+  private final Map<ReportRuleKey, RuleReport> ruleReportByRuleKey = new HashMap<>();
 
   private List<TrackedIssue> issues = new ArrayList<>();
-  private Map<Integer, List<TrackedIssue>> issuesPerLine = Maps.newHashMap();
-  private Map<Integer, List<TrackedIssue>> newIssuesPerLine = Maps.newHashMap();
-  private Map<Rule, AtomicInteger> issuesByRule = Maps.newHashMap();
-  private Map<RulePriority, AtomicInteger> issuesBySeverity = Maps.newHashMap();
+  private Map<Integer, List<TrackedIssue>> issuesPerLine = new HashMap<>();
+  private Map<Integer, List<TrackedIssue>> newIssuesPerLine = new HashMap<>();
+  private Map<Rule, AtomicInteger> issuesByRule = new HashMap<>();
+  private Map<RulePriority, AtomicInteger> issuesBySeverity = new EnumMap<>(RulePriority.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "RulePriority"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)